tests/IssueTest.php

```diff
@@ -81,24 +55,7 @@ public function testIssueJsonSerialize_nullableDescriptionSuccess()
         $this->assertNull($result['summary']);
```

Comment:
You should _actually_ make sure that the output matches an expected output. The expected output is what the frontend will receive. This is the whole point of testing the JSON encoding - that the outputs match what the frontend expects.

So you should `json_encode` the issue and an expected result issue and see if those match, like the other tests.

Reply:
done